src/main/java/org/broadinstitute/hellbender/tools/ApplyBQSRUniqueArgumentCollection.java

```diff
@@ -19,6 +23,25 @@
     @Argument(fullName="quantize_quals", shortName = "qq", doc = "Quantize quality scores to a given number of levels", optional=true)
     public int quantizationLevels = 0;
 
+
+    /**
+     * Static quantized quals are entirely separate from the quantize_qual option which uses dynamic binning.
+     * The two types of binning should not be used together.
+     */
+    @Advanced
```

Comment:
Advanced isn't currently honored in any way in the arg parser.  Open a ticket to support it if you think it would be useful to display those arguments separately.